File: backend/dto/Prestamo.go
package dto

import "time"

type PrestamoDTO struct {
	Id_libro                 int
	Id_bibliotecaria         int
	Nro_matricula_estudiante string

	Fecha_prestamo   time.Time
	Fecha_devolucion time.Time

	Estado string
}

type FinPrestamoDTO struct {
	Id_prestamo   int
	Id_libro      int
	Id_estudiante int
}

File: frontend/src/models/Prestamo.ts
interface Prestamo {
  Id: number;
  // Id_libro: number;
  Id_estudiante: number;
  Id_bibliotecario: number;

  Fecha_prestamo: Date;
  Fecha_devolucion: Date;

  Estado: string;
}

interface FinPrestamoDTO {
  Id_prestamo: number;
  Id_libro: number;
  Id_estudiante: number;
}

export type { Prestamo, FinPrestamoDTO };
